modules/bundles/get_started.ipynb

```diff
@@ -517,12 +517,10 @@
    ]
```

Comment:
<p>Here I suggest to add:</p><p><code></code><code></p><p>If the escape character <code>\</code> is needed in your shell, <code>--device "\$torch.device('cuda:1')"</code> should be used. </p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/649/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
updated according to your comments.

Thanks.